pom.xml

```diff
@@ -14,6 +14,19 @@
 		<!--module>pic-sure-api-wildfly</module-->
 		<module>pic-sure-util</module>
 	</modules>
+	<repositories>
+		<repository>
+			<id>central</id>
+			<url>https://repo1.maven.org/maven2</url>
+		</repository>
```

Comment:
This may not be necessary. When configuring the github repo in maven settings.xml, this had to be here otherwise maven would try github first for every single dependency. But I think this may no longer be needed.